components/com_content/tmpl/category/blog.php

```diff
@@ -41,8 +42,8 @@
 
 	<?php if ($this->params->get('show_category_title', 1)) : ?>
 	<<?php echo $htag; ?>>
-		<?php echo $this->category->title; ?>
-	</<?php echo $htag; ?>>
+ 		<?php echo $this->category->title; ?>
+ 	</<?php echo $htag; ?>>
```

Comment:
```suggestion
		<?php echo $this->category->title; ?>
	</<?php echo $htag; ?>>
```

removes extra leading space